src/test/java/core/basesyntax/shopoperations/SupplyTest.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.shopoperations;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+public class SupplyTest {
+
+    @Test
+    public void calculate_ValidAmount_Ok() {
+        assertEquals(50, new Supply().calculate(35, 15));
```

Comment:
what if we supply "-15"?

Reply:
No need to test because i have DataValidator which carrying about this.